### PR TITLE
direct new contributors

### PR DESCRIPTION
**Pull request by titaniumbones**
just brief language to make sure contributors know how to get to sentry and webapp.

We should have another line for new endorsers, but I'm not sure what we want to do with them, so...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### <em>Community Patterns for the Decentralized Web</em>
 
+We're building a better future for data [on our beta platform](https://archivers.co). come join us!
```
Comment (dcwalk) [inline README.md]: Wondering if this should be our tagline below?

**Comment by dcwalk** [comment]
@flyingzumwalt could we get a quick review? otherwise I'll aim to merge in over the weekend

File: README.md (hunk 1 above)
Comment (flyingzumwalt) [reply README.md]: I'm strongly averse to presenting Data Together as an organization that's creating yet another competing platform in an an increasingly crowded space.  That's not actually what we're doing. Data Together has the capacity to be a rallying point for everyone who thinks we should build the new web around patterns of shared possession and communities coordinating to possess the data they rely on. As part of that, the platform we're building is one example of how to do this but it is not trying to the the ONE solution. It's solving known needs (ie. orgs like EDGI need to give their contributors ways to participate in Harvesting, Storing, Monitoring, etc), and it's addressing them by taking a novel approach to the intersection between decentralized protocol, community organizing, and the role of institutions like libraries. It's a practical solution that we intend to maintain over time, and it's a reference example, but that's in service of the main purpose of Data Together, which is to get everyone strongly aligned around the importance of grappling with the problems of collective action on the decentralized web.  If we don't grapple with those problems, and if the many projects in this space don't align around tackling those problems, the next version of the web will end up being even more prone to monopolies and exploitation than the current HTTP web. Data Together is an opportunity to get ahead of that.

Comment (flyingzumwalt) [reply README.md]: Possibly rewrite to:
> We're building a better future for data. The [beta platform at https://archivers.co](https://archivers.co) is one example of where we're going. Come join us!

I will rebase my changes and submit them in a PR after we merge this one, to keep things clean.

Comment (dcwalk) [reply README.md]: that would be awesome @flyingzumwalt 🎉

Comment (titaniumbones) [reply README.md]: Added a commit addressing this @flyinzumwalt.

**Pull request merged**